docs/src/main/paradox/index.md

```diff
@@ -5,6 +5,8 @@ This guide introduces Akka by describing the Scala version of the Hello World ex
 
 Actors are the unit of execution in Akka. The Actor model as an abstraction that makes it easier to write correct concurrent, parallel and distributed systems. The Hello World example illustrates Akka basics. Within 30 minutes, you should be able to download and run the example and use this guide to understand how the example is constructed. This will get your feet wet, and hopefully inspire you to dive deeper into the wonderful sea of Akka!
 
+After trying this example the comprehensive [Getting Started Guide](http://doc.akka.io/docs/akka/2.5/scala/guide/introduction.html) is a good next step to continue learning more about Akka.
+
```

Comment:
The link to the getting started guide was only at the end, rather hidden.
It's a fine balance, we don't want too many links away from this guide, but somehow I would like to promote the full Getting Started guide also.